rosdep/python.yaml

```diff
@@ -6965,6 +6965,13 @@ python3-retrying:
     pip:
       packages: [retrying]
   ubuntu: [python3-retrying]
+python3-rocker:
+  alpine: [rocker]
+  debian: [python3-rocker]
+  fedora: [python3-rocker]
+  gentoo: [dev-util/rocker]
```

Comment:
I don't see this available on gentoo: https://packages.gentoo.org/packages/search?q=rocker